security/openshift-security/confluent-platform-with-defaultSCC.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: platform.confluent.io/v1beta1
+kind: Zookeeper
+metadata:
+  name: zookeeper
+  namespace: operator
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cp-zookeeper-operator:6.0.0.0
+    init: confluentinc/cp-init-container-operator:6.0.0.0
+    pullSecretRef:
+      - confluent-registry
```

Comment:
seems like copy-paste error, not required as we are pulling image from dockerhub